index.html

```diff
@@ -0,0 +1,201 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="description" content="recipe-app">
+        <meta name="keywords" content="HTML, CSS, JavaScript, recipe-app">
+        <meta name="viewport" content="width=device-width, initial scale=1.0">
+        <meta name="author" content="Joseph Mitiny">
+        <link rel="stylesheet" href="styles.css">
+        <!-- <script src="https://kit.fontawesome.com/a076d05399.js" crossorigin="anonymous"></script> -->
+        <script src="https://kit.fontawesome.com/a462069d14.js" crossorigin="anonymous"></script>
+        <title>recipe-app</title>
+    </head>
+
+    <body>
+        <h2>Welcome!! Check Below for todays Delicacy</h2>
+
+        <div class="main-header">
+            <div>
+                <input type="text" name="search" required placeholder="Search"/>
+            </div>
+            <div>
+                <input type="text" name="filter" required placeholder="Filter"/>
+            </div>
+            <div>
+                <input type="text" name="favourites" required placeholder="Favourites"/>
```

Comment:
this should be a button , so that when the user clicks inafilter inampea favourites zake

Reply:
> this should be a button , so that when the user clicks inafilter inampea favourites zake

i have made it a button, pending js code to make it more interactive.